src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsOnSpecialOasis.java

```diff
@@ -0,0 +1,81 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+/**
+ * https://www.google.com/dfp/5441#delivery/LineItemDetail/lineItemId=126608052
+ */
+
+public class TestAdsOnSpecialOasis extends TemplateNoFirstLoad {
+  final static String PROJECT43_TEST_LINE_ITEM_ID = "271491732";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)